libraries/AP_HAL_ESP32/boards/esp32s3empty.h

```diff
@@ -26,70 +74,111 @@
 
 #define HAL_ESP32_BOARD_NAME "esp32s3empty"
 
-#define HAL_ESP32_RMT_RX_PIN_NUMBER GPIO_NUM_14
+#define HAL_ESP32_RMT_RX_PIN_NUMBER GPIO_NUM_1
+
+//#define HAL_INS_DEFAULT HAL_INS_NONE
+
 
-// no sensors
-#define HAL_INS_DEFAULT HAL_INS_NONE
+#define AP_INERTIALSENSOR_ENABLED 1
+
+#define HAL_INS_DEFAULT HAL_INS_MPU9250_I2C
+#define HAL_INS_MPU9250_NAME "MPU9250"
+
+// BARO choices:
+#define HAL_BARO_DEFAULT HAL_BARO_BMP280_I2C
+#define HAL_BARO_BMP280_NAME "BMP280"
+
+// IMU probing:
+#define HAL_INS_PROBE_LIST PROBE_IMU_I2C(Invensense, 0, 0x68, ROTATION_YAW_180)
```

Comment:
the 'xxxempty' board/s are a bit special, and are designed to work with no sensor/s attached, and this looks like its changing that.?